src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin: 0 5px;
+  padding: 0 5px;
+  position: relative;
+}
+
+.nav__item:first-child {
+  padding-left: 0;
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+  margin-right: 0;
+}
+
+.nav__link {
+  height: 60px;
+  line-height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  cursor: pointer;
+}
+
+.is-active1 {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  text-decoration-line: underline;
+  text-decoration-style: solid;
+  text-decoration-color: #00acdc;
+  text-decoration-thickness: 3px;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  top: 56px;
+  left: 50%;
+  transform: translateX(-51%);
+  height: 4px;
+  width: 37px;
```

Comment:
The element should not have set fixed width. It should be 100% width of the link.

Reply:
Done